### ISSUE TITLE
Bug: Fix Progressive Web App Errors Highlighted By Lighthouse

### ISSUE THREAD

**Issue by wordshaker** [DDDEastMidlandsLimited/dddem-web#216]
### Describe the feature

The site currently has a number of errors when checked when checking for the Progressive Web App standards in Lighthouse.

The details of improvements that can be made are shown in the lighthouse report

![image](https://user-images.githubusercontent.com/13496774/110368263-b5207e80-8040-11eb-9922-97844927de89.png)

### To Get Lighthouse Score

- Open the site in Chrome
- Open developer tools
- Go to the Lighthouse tab in the dev tools
- Run.

### Advantages

- Improving the Progressive Web App of the site

#### Acceptance Criteria

Functional and non-functional criteria that should be met by this work

- [ ] Lighthouse errors for Progressive Web App is decreased

**Comment by paolapog** [comment]
Hello @wordshaker ! Is this still an issue? If it is so, I'd like to help you in someway. I’m a first-time contributor but I already worked with PWA's in my actual job. If it is okay for you, assign to me the issue, thanks!

**Comment by wordshaker** [comment]
Hi @paolapog. This is still and issue and it's all yours 😄

**Comment by wordshaker** [comment]
Thank you for your contribution @paolapog

**Comment by wordshaker** [comment]
![image](https://user-images.githubusercontent.com/13496774/111886139-eb0c1e00-89c3-11eb-8178-480c5305b5cb.png)
